app/src/main/java/uk/co/ashtonbrsc/intentexplode/SettingsActivity.java

```diff
@@ -28,6 +28,7 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.settings);
+        setTitle(getTitle() + " - " + SettingsUtil.getAppVersionName(this));
```

Comment:
I don't think the app version is so important that it should be put in the Action Bar.